state/allwatcher.go

```diff
@@ -1826,6 +1861,65 @@ func (ctx *allWatcherContext) getInstanceData(id string) (instanceData, error) {
 	return getInstanceData(ctx.state, id)
 }
 
+func (ctx *allWatcherContext) assignedMachineID(u *Unit) (string, error) {
+	// c.f. unit.AssignedMachineId.
```

Comment:
Remove comment?

Reply:
I'll expand on what I want this comment to mean.